framework/src/wifi_manager/wifi_profile.c

```diff
@@ -262,13 +262,13 @@ wifi_utils_result_e wifi_profile_read(wifi_manager_ap_config_s *config)
 	if (ret != SEE_OK) {
 		return WIFI_UTILS_FILE_ERROR;
 	}
+	nvdbg("readed data len(%d)\n", readlen);
```

Comment:
Shouldn't it be _**read**_ instead of **_readed_**?

Reply:
It is better to print unsigned int as %u.